### ISSUE TITLE
RFE: list-units --filter=myservice and info-units

### ISSUE THREAD

**Issue by dougsland** [engelmi/hirte#286]
## Describe the bug

This is an RFE to add `--filter` option into `list-units` command.  Today grep can be used as workaround.

Example(s):
```
# hirtectl list-units --filter=service1
# hirtectl list-units --filter=service1, service2
```

Please also add a `command alias for this part of the code` called `hirte info-units`, example:
```
# hirtectl info-units service1
# hirtectl info-units service1, service2
```

**Comment by rhatdan** [comment]
I think you mean hirtectl, not hirte.

**Comment by rhatdan** [comment]
I had assumed 
hirtectl list-units service1 service2 would work. Along with
hirtectl list-unitx service1@node1 service2 would work.
In this case it would list only service1 on node1, but service2 on any node where it exists.

**Comment by dougsland** [comment]
> I think you mean hirtectl, not hirte.

correct Dan, thanks for pointing it.

**Comment by dougsland** [comment]
Sounds good too @rhatdan. Not implemented yet.

```
# hirtectl list-units boot.mount
Failed to issue method call: Unknown object '/org/containers/hirte/node/boot_2emount'.
Call to manager failed: Invalid request descriptor

# hirtectl list-units boot.mount@node1
Failed to issue method call: Unknown object '/org/containers/hirte/node/boot_2emount_40node1'.
Call to manager failed: Invalid request descriptor
```

```
# hirtectl list-units | grep boot.mount
control             |boot.mount                                                 | inactive|     dead
node1               |boot.mount                                                 | inactive|     dead
```

**Comment by rhatdan** [comment]
Ok so the --filter makes some sense then. Since third option is considered a node specifier.

**Comment by engelmi** [comment]
@dougsland Regarding the alias `info-units` for `list-units`:
What  would be the benefit of this alias? Since `hirtectl` is basically systemctl for multi-node we try to align the commands as much as possible - therefore the `list-units` command. I'd try to keep the CLI as lean as possible.

**Comment by rhatdan** [comment]
I agree we should mimick systemctl as much as possible in hirtectl. Easier to transfer skills.

**Comment by dougsland** [comment]
> I agree we should mimick systemctl as much as possible in hirtectl. Easier to transfer skills.

@engelmi  @rhatdan answered, all good. Moving forward :)

**Comment by engelmi** [comment]
Implemented in #311. Closing.
